qiskit/synthesis/two_qubit/local_invariance.py

```diff
@@ -65,7 +65,7 @@ def two_qubit_local_invariants(U: np.ndarray) -> np.ndarray:
     # to better equate to the Weyl chamber coordinates (c0,c1,c2)
     # and explore the parameter space.
     # Also do a FP trick -0.0 + 0.0 = 0.0
-    return np.round([G1.real, G1.imag, G2.real], 12) + 0.0
+    return np.round([G1.real, G1.imag, G2.real], 9) + 0.0
```

Comment:
```suggestion
    return [G1.real, G1.imag, G2.real]
```
Or just drop this unnecessary wrapper completely. Why are you throwing away info and reducing the precision further. I know this was pre existing behavior but it makes much more sense to go the other way and remove rounding completely here - i don't understand why we want to throw away information.